src/components/home-page/hero-section/hero-section.vue

```diff
@@ -0,0 +1,92 @@
+<script>
+    import { defineComponent } from 'vue';
+
+    import BaseButton from '@/components/global/base-button.vue';
+
+    export default defineComponent({
+        name: 'HeroSection',
+        components: {
+            BaseButton,
+        }
+    });
+
+</script>
+
+<template>
+    <section class="hero-section">
+        <div class="hero-text-container">
+            <h1 class="hero-section-heading">
+                Welcome to the <br/> best cinema in Monterail
+            </h1>
+            <p class="hero-section-paragraph">
+                Best movies, best experiences, best people.
+                
+                And all of that doesn't exist. Yikes.
+            </p>
+            <BaseButton class="hero-action-button" :is="'a'" label="Book a ticket" colorScheme="redFill" size="56" />
+        </div>
+        <div class="hero-image-container">
+            <img class="hero-image" src="@/assets/images/hero-section-image.png" />
+        </div>
+    </section>
+</template>
+
+<style lang="scss">
```

Comment:
You might want to consider using `<style scoped>` here.
This way we can not worry about class names leaking out by accident 😄 

Related: https://vuejs.org/style-guide/rules-essential.html#use-component-scoped-styling

Reply:
Was using the `style scoped` attribute in the first days of the bootcamp and must have forgotten about it - thanks for the reminder!
Same as with the above - will update in all files after rebase.